concepts/philosophy.md

```diff
@@ -1,5 +1,9 @@
 # Philosophy
 
+```{note}
+Can probably be reworked so that core principles appear at beginning of manifesto
+```
+
```

Comment:
so, I'm not 100% sure we should merge the manifesto and the philosophy part together, I like having them separate because they represent 2 different things:
- the manifesto describes "what we do" and acts like the statutes of the collective
- the philosophy section describes "why we do it" and why we believe it will work

It doesn't mean we can't have links and backrefs between the 2 documents, of course.

Another reason to keep them separated is that the manifesto should be clear and concise (inasmuch as possible), while the philosophy section can be expanded and end up being a document of significant length.